pkg/vault/package.json

```diff
@@ -18,7 +18,7 @@
   ],
   "scripts": {
     "build": "yarn compile",
-    "compile": "hardhat compile && rm -rf artifacts/build-info",
```

Comment:
do *NOT* toss away our precious enhanced build-info json files.
![image](https://user-images.githubusercontent.com/380950/149637709-5d291231-2ab1-4f26-8c15-5951e80eb566.png)